test/cli/autogen-subclasses/a.rb

```diff
@@ -1,7 +1,10 @@
 # typed: true
 
 module Opus
-  class DupChild < Parent; end
+  class DupChild < Parent
+    def defines_behavior
+    end
+  end
```

Comment:
Is it important that `DupChild` is no longer defined twice in this test?

Reply:
It is not important for the purposes of this PR, I will try to put that part back.

Reply:
It does occur twice; I just had to move the second "definition" to `dup_child.rb`.  I don't think anything needs to be changed here.